umake/frameworks/ide.py

```diff
@@ -473,3 +474,115 @@ def post_install(self):
                                                      categories=categories))
         if not self.was_in_arduino_group:
             UI.delayed_display(DisplayMessage(_("You need to logout and login again for your installation to work")))
+            
+class BaseNetBeans(umake.frameworks.baseinstaller.BaseInstaller):
+    """The base for all Netbeans installers."""
+    
+    BASE_URL = "http://download.netbeans.org/netbeans/"
+    executable = "nb/netbeans"
+
+    def __init__(self, category, flavour=""):
+        """ The constructror.
+        @param category The IDE category.
+        @param flavour The Netbeans flavour (plugins bundled).
+        """
+        #add a separator to the string, like -cpp
```

Comment:
Please add a space between # and start of the comment (if I didn't do it everywhere, blame me :p)